src/services/balancer/pools/exits/handlers/generalised-exit.handler.ts

```diff
@@ -60,15 +62,30 @@ export class GeneralisedExitHandler implements ExitPoolHandler {
     const signerAddress = await signer.getAddress();
     const slippage = slippageBsp.toString();
 
-    this.lastExitRes = await balancer.pools.generalisedExit(
-      this.pool.value.id,
-      evmAmountIn.toString(),
-      signerAddress,
-      slippage,
-      signer,
-      SimulationType.Tenderly, // TODO: update to use VaultModel + Static (see SDK example for more details)
-      relayerSignature
-    );
+    // Static call simulation is more accurate than VaultModel, but requires relayer approval and
+    // account to have enough BPT balance
+    const simulationType: SimulationType =
+      bptInValid && !approvalActions.length
```

Comment:
Seems like this `!approvalActions.length` will always be true. So should we just be using the VaultModel for all exits?

Reply:
I'm not sure I understand. Only approval action for exits is currently the relayer approval, so we need to check for that.

VaultModel = TS math, are you suggesting it is accurate enough to use always?